client/onboarding/steps/loading.tsx

```diff
@@ -67,6 +67,7 @@ const LoadingStep: React.FC< Props > = () => {
 		const resultUrl = addQueryArgs( connectUrl, {
 			self_assessment: fromDotNotation( data ),
 			progressive: isEligible,
+			source: 'onboarding',
```

Comment:
Maybe be more specific about the source like `onboarding-wizard`

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4